src/main/org/nlogo/nvm/LetBinding.java

```diff
@@ -4,9 +4,9 @@
 
 import org.nlogo.api.Let;
 
-strictfp class LetBinding {
-  final Let let;
-  Object value;
+strictfp public class LetBinding {
+  public final Let let;
+  public Object value;
```

Comment:
> @TheBizzle , what were you saying about grad students going around making all the fields public?

The great irony here is that @qiemem himself was the one saying that.  :laughing: